tests/spawner/test_spawner.py

```diff
@@ -85,13 +85,17 @@ def test_args(self):
         self.assertIn("--config-file={}".format(path), args)
         self.assertIn("--base-urlpath=/", args)
 
+    def test_args_without_config_file_path(self):
+        args = self.spawner.get_args()
+        self.assertIn("--proxy-api-url=http://127.0.0.1:12345/foo/bar/", args)
+        self.assertFalse(any("--config-file=" in arg for arg in args))
```

Comment:
Yes, but I think that loops over the content of`args` twice?  `any` returns `True` for the first `True` (although if the test should pass, it loops over the whole thing, once).